FaynaTown.Tennis.Planner.Bot.Core/Providers/JsonExtensionProvider.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.IO;
+
+namespace FaynaTown.Tennis.Planner.Bot.Core.Providers;
+internal class JsonExtensionProvider : IFileExtensionProvider {
```

Comment:
Little format refactoring suggestion

```suggestion
namespace FaynaTown.Tennis.Planner.Bot.Core.Providers;

internal class JsonExtensionProvider : IFileExtensionProvider {
```

Reply:
Done